src/loaders/compute_tools/tool_common.py

```diff
@@ -575,7 +579,7 @@ def write_fatal_tuples_to_dict(fatal_tuples: List[FatalTuple], output_dir: Path)
 
     fatal_error_path = os.path.join(output_dir, loader_common_names.FATAL_ERROR_FILE)
     with open(fatal_error_path, "w") as outfile:
-        outfile.dump(fatal_dict, outfile)
+        json.dump(fatal_dict, outfile)
```

Comment:
Should this have an indent? Otherwise I'd assume it's really hard to read

Reply:
I think i do have an indent within context manager. You mean have a blank line between line 580 and 581?

Reply:
:+1: